test/integration/targets/meraki_organization/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+- name: List all organizations
+    meraki_organiztaion:
```

Comment:
Indentation is wrong here, I believe this should be:
```yaml
- name: List all organizations
  meraki_organiztaion:
    auth_key: abc123
    status: query
  delegate_to: localhost
```
Which is causing the yamllint issue.